Dockerfile

```diff
@@ -94,20 +78,38 @@ COPY --link usr/local/sbin /usr/local/sbin/
 COPY --link usr/local/share/deskpro /usr/local/share/deskpro/
 
 RUN <<EOT
-    ln -s /etc/php81 /etc/php
-    ln -s /usr/sbin/php-fpm81 /usr/sbin/php-fpm
+    set -e
```

Comment:
Is the removal of symlink to `/etc/php` a conscious decision? Would it not make it (slightly) easier to upgrade php in the future?

Reply:
@chrislrobinson The supported way to add custom php.ini is only via `/deskpro/config/php.d`, so the link itself only matters from our side I think.

And from _our_ side, I think having absolute paths everywhere makes it easier to see what actually changes if we update PHP. Makes it a bit easier to see scope of changes and mistakes a bit more obvious to see (e.g. rspec tests will fail when files don't exist).

But the main reason I removed it is because the link itself wasn't backwards compat because debian slim separates php.ini for cli vs fpm